### ISSUE TITLE
Links on index & 404 page

### ISSUE THREAD

**Issue by Leo-Mu** [muan/hello-world#101]
## Why not link to everyone's file in the `humans` directory?
Well. Now on the index page, we can only see the link to human.txt and some links to everyone's GitHub profile. If I want to see a page someone created in `humans` directory, it will be hard to find it. Why not write a piece of JavaScript to display links to pages in the `humans` directory rather than GitHub profiles.
## Do you need a 404 page?
I found there is no a particular 404 page. If the URL is wrong, it will display the boring GitHub 404 page.

**Comment by SubhrajyotiSen** [comment]
@muan This sounds like a good idea

**Comment by zwirektech** [comment]
I agree about the files in humans directory. It has been discussed a little [in my first PR](https://github.com/muan/hello-world/pull/90#issuecomment-295846997).
Since the picture is a link to the user's profile, there could be a filename under the picture as a link to the file in humans folder.
However, the filename doesn't have to be the same as a username in human.txt
Actually it's not the same in many cases.
I think just a list of humans folder content with links to individual files would be ok.

The 404 could be another issue...

**Comment by aaossa** [comment]
@muan any opinions on this? 🤔

**Comment by muan** [comment]
> Why not link to everyone's file in the `humans` directory?

We could. The only problem is we didn't restrict what kind of file people can put in the repo. It could be `.md` or `.html` or `.txt` etc. I didn't want having to write HTML to be a blocker. But perhaps that's not that big of an issue?

*Or* we could do something clever with JS `fetch` to test what type of file by a name exist and then redirect people to that URL. 

> Do you need a 404 page?

404 would be good. I can go hand in hand with that ^ redirection idea.

**Comment by leewp14** [comment]
Hi there!
I made a very very very simple list of humans, testing indicates that txt, md and html is ok to open directly, except for those without a file extension. 
Here I attached the html file, don't know whether it can be used :smile: 

[test.html.zip](https://github.com/muan/hello-world/files/2209912/test.html.zip)
